packages/calcite-components/src/components/slider/slider.tsx

```diff
@@ -995,15 +995,14 @@ export class Slider
    * @private
    */
   private getHostOffset(leftBounds: number, rightBounds: number): number {
-    const hostBounds = this.el.getBoundingClientRect();
-    const buffer = 7;
```

Comment:
At first I wasn't sure about removing this buffer, but now that I look at the behavior, there's no more jumpiness in the label offset positioning any more.  I think the original intention to this buffer was to keep the label lined up exactly with the edge boundary of the track, but I like this better!  👍